src/som/interpreter/Frame.java

```diff
@@ -171,6 +171,10 @@ public void setStackElement(final int index, final SAbstractObject value) {
     stack[getStackPointer() - index] = value;
   }
```

Comment:
I would have assumed this file to be disappearing?

Reply:
Yes, but I need to take care of the Shell first (still not using truffle frames). - it'll be gone eventually

Reply:
Done with the new commits